_templates/press.html

```diff
@@ -19,11 +19,9 @@
   quotejeremyliew: "The potential for disruption is enormous."
   quotewencescasares: "Right now Bitcoin feels like the Internet before the browser."
   quotetylerwinklevoss: "We have elected to put our money and faith in a mathematical framework that is free of politics and human error."
-  quotemaxkeiser: "It's the cheapest way to move money around."
```

Comment:
NACK 

It's a quote/citation, not written in stone. Bitcoin is one of the cheapest ways to move money around.